### PR TITLE
Update facturation documents

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

File: frontend/src/components/Logo/LogoFooter.jsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
+import { HashLink } from 'react-router-hash-link';
 
 const LogoFooter = () => (
   <div className="fr-footer__brand">
-    <a className="fr-logo" href="/" title="République française">
+    <HashLink className="fr-logo" to="/" title="République française">
```
Comment (sandrica89) [inline frontend/src/components/Logo/LogoFooter.jsx:6]: Shouldn't this point to MESRI website?

**Comment by sandrica89** [comment]
PDF facturation file has an empty table while DOCX and ODT files have 2 completes rows. Is this expected? 
I would keep the tables empty to avoid any confusion.

File: frontend/src/components/Logo/LogoFooter.jsx (hunk 1 above)
Comment (desoindx) [reply frontend/src/components/Logo/LogoFooter.jsx:6]: i'll ask in the chat and create a ticket if needed

**Comment by desoindx** [comment]
> PDF facturation file has an empty table while DOCX and ODT files have 2 completes rows. Is this expected?
> I would keep the tables empty to avoid any confusion.

image

**Pull request merged**